packages/techdocs-common/src/stages/publish/local.ts

```diff
@@ -52,17 +51,14 @@ try {
  * called "static" at the root of techdocs-backend plugin.
  */
 export class LocalPublish implements PublisherBase {
-  private readonly config: Config;
-  private readonly logger: Logger;
-  private readonly discovery: PluginEndpointDiscovery;
-
   // TODO: Use a static fromConfig method to create a LocalPublish instance, similar to aws/gcs publishers.
   // Move the logic of setting staticDocsDir based on config over to fromConfig,
   // and set the value as a class parameter.
   constructor(
-    config: Config,
-    logger: Logger,
-    discovery: PluginEndpointDiscovery,
+    // @ts-ignore
```

Comment:
is this necessary?

Reply:
The `config` is not being used anymore in this publisher and so Typescript kept complaining.